.github/Compiler Guide.md

```diff
@@ -0,0 +1,74 @@
+
+The Compiler, aside from just merely putting the files into the correct places in the Sector File, is also capable of validating input. This makes it incredibly useful, bordering on essential, for developing 'correct' sector files that work as intended and catching very understandable human error when dealing with Euroscope's esoteric syntax requirements. For example, coordinates entered in the wrong format will produce the following message:
+![image](https://user-images.githubusercontent.com/14115426/208726651-1ff76baa-a197-4120-acb6-1e9f7061d013.png)
+
+We will explore understanding these error messages later.
+
+The purpose of this guide is to enable contributors to get the compiler working locally to encourage local development (rather than uploading PRs that then fail the compiler checks). Usage of the compiler locally also facilitates working within the structure of the Sector File repository while developing, meaning contributors do not need to edit extremely large and cumbersome .sct and .ese files, and avoids changes being made that then get lost in those large files.
+
+This guide will make the following assumptions, as these seem to be the most common circumstances under which contributors work:
+
+- You are using Windows 10/11
+- You are using GitHub Desktop
+
+It is very much possible to use the Compiler under completely contrary circumstances, i.e. you are not using Windows 10/11, and not using GitHub desktop (e.g. using Git standalone). If this applies, and would like guidance on how to use the Compiler, do put a message in #sector_file_development, as it is likely someone can help. On the other hand, if you are happy using the compiler, then the author would be grateful if you could extend this guide for those circumstances through a PR - every contribution helps!
+
+## Downloading the Compiler
+
+To download the compiler, first navigate to the [Compiler releases page](https://github.com/VATSIM-UK/sector-file-compiler/releases). Then, download the latest release - marked with a green label of 'Latest and will be at the top of the list. You will want to click the 'cli-windows-x64' button - this should trigger a download:
+![image](https://user-images.githubusercontent.com/14115426/208726535-de2d20d0-0b00-45fd-b2f8-9d79974dc850.png)
+
+Once this has downloaded, open your Downloads folder, and rename the 'cli-windows-x64' file to 'cli-windows-x64.exe'. This allows Windows to run it. It then needs to be moved by cutting-and-pasting it into your local repository folder. If you are using GitHub Desktop, this will be in your Documents->GitHub->UK-Sector-File. It needs to go into that final folder, i.e. Documents/GitHub/UK-Sector-File/cli-compiler-x64.exe. Don't worry about Git picking this up - all .EXEs are automatically excluded, as are the filenames.

```

Comment:
Mine is somewhere else, I think you can choose where you want it when you set up the fork, I have it in `C:\Users\peter\OneDrive\Documents\Flying\VATSIM\UK SF Dev\GitHub_UK-Sector-File`.
```suggestion
Once this has downloaded, open your Downloads folder, and rename the 'cli-windows-x64' file to 'cli-windows-x64.exe'. This allows Windows to run it. It then needs to be moved by cutting-and-pasting it into your local repository folder. If you are using GitHub Desktop, this will probably be in your Documents->GitHub->UK-Sector-File. It needs to go into that final folder, e.g. Documents/GitHub/UK-Sector-File/cli-compiler-x64.exe. Don't worry about Git picking this up - all .EXEs are automatically excluded, as are the filenames.

```